docs/en/stack/ml/df-analytics/ml-feature-importance.asciidoc

```diff
@@ -0,0 +1,35 @@
+[role="xpack"]
+[[ml-feature-importance]]
+=== Understanding results: {feat-imp}
+
+{feat-imp-cap} is calculated for supervised {ml} methods such as 
+<<dfa-regression,{regression}>> and <<dfa-classification,{classification}>>. 
+This value provides further insight into the results of a {dfanalytics-job} and 
+therefore helps interpret these results. As we mentioned, there are multiple 
+features of a data point that are analyzed during {dfanalytics}. These features 
+are responsible for a particular prediction to varying degrees. {feat-imp-cap} 
+shows to what degree a given feature of a data point contributes to the 
+prediction. The feature importance value of a feature can be either positive or 
+negative depending on its effect on the prediction. If the feature reduces the 
+prediction value, the value is negative. If the feature increases the 
+prediction, the {feat-imp} value positive. The magnitude of the {feat-imp} 
+value shows how significantly the feature affects the prediction both locally 
+(for a given data point) or generally (for the whole data set).
+
+{feat-imp-cap} in the {stack} is calculated using the SHAP (SHapley Additive 
+exPlanations) method as described in
+https://papers.nips.cc/paper/7062-a-unified-approach-to-interpreting-model-predictions.pdf[Lundberg, S. M., & Lee, S.-I. A Unified Approach to Interpreting Model Predictions. In NeurIPS 2017].
+
+By default, {feat-imp} values are not calculated. To generate this information, 
```

Comment:
> By default, feature importance values are not calculated.

I think this is only true of the API.  When I use Kibana, it seems to be consistently using a default value of 2.

Reply:
I amended the text accordingly.